packages/vanilla/src/components/horizontalrule/README.md

```diff
@@ -0,0 +1,91 @@
+# Horizontal Rule
+
+> The horizontal rule component is to be utilized within IBM.com for thematic
+> breaks within the content of the page.
+
+## Getting started
+
+Here's a quick example to get you started:
+
+### ES6 Template Literal
+
+```javascript
+import { horizontalruleTemplate } from '@carbon/ibmdotcom-vanilla';
+import '@carbon/ibmdotcom-styles/scss/components/horizontalrule/index.scss';
+
+console.log(
+  horizontalruleTemplate({
```

Comment:
Might I suggest something like adding it into the page instead of consoling?

```javascript
document.body.innerHTML = horizontalruleTemplate();
```

Reply:
since it's a README I don't really have much of a preference. At least for the Services and Utilities `@example` sections, we use console.log.

Adding @kennylam @annawen1 @andysherman2121 @sannminwin for any opinions on this.